i18n/zh/docusaurus-plugin-content-docs/current/welcome/why-kubeedge.md

```diff
@@ -14,7 +14,7 @@ KubeEdge 的优势主要包括：
 
 - **边缘计算**
 
-  借助在 Edge 上运行的业务逻辑，可以让本地生成的数据，进行大量数据处理操作并对其进行保护。这样可以减少边缘和云之间的网络带宽需求和消耗，提高响应速度，降低成本并保护客户的数据隐私。
+  借助在 Edge 上运行业务应用，可以在产生数据的边端本地存储和大量数据的预处理。这样可以减少边缘和云之间的网络带宽需求和消耗，提高响应速度，降低成本并保护客户的数据隐私。
```

Comment:
```suggestion
  借助在 Edge 上运行业务应用，可以在产生数据的边端存储和处理大量数据。这样可以减少边缘和云之间的网络带宽需求和消耗，提高响应速度，降低成本并保护客户的数据隐私。
```

Reply:
Thank you for your advice, I have made the corresponding changes.